cv32e40s/tb/uvmt/uvmt_cv32e40s.flist

```diff
@@ -63,10 +63,12 @@ ${TBSRC_HOME}/core/tb_riscv/riscv_gnt_stall.sv
 ${TBSRC_HOME}/core/tb_riscv/riscv_rvalid_stall.sv
 ${TBSRC_HOME}/core/tb_riscv/riscv_random_interrupt_generator.sv
 
-${DV_UVMT_PATH}/uvmt_cv32e40s_interrupt_assert.sv
```

Comment:
We are creating a large set of Assertion files.  This is great!  Does it make sense to package them separately?

Reply:
I've thought the same.

How do you envision it?
Simply create a new directory within "tb/uvmt/"?
Create a new directory like e.g. "tb/asserts/"?
Separate these assertion files into a new ".flist" file?

Can we do it in a separate PR?

Reply:
I was thinking something like `tb/asserts`, because they are not really a "UVM-thing" and ideally these assertions can be (are being???) used in both simulation and formal.

Yes, this can be a separate PR.

Reply:
I agree that it makes sense to put these in a common location, tb/asserts makes sense to me